crates/apollo-mcp-server/src/runtime/logging.rs

```diff
@@ -44,17 +49,13 @@ impl Default for Logging {
             level: defaults::log_level(),
             path: None,
             rotation: defaults::default_rotation(),
+            format: default_format(),
         }
     }
 }
 
 type LoggingLayerResult = (
-    Layer<
-        tracing_subscriber::Registry,
-        tracing_subscriber::fmt::format::DefaultFields,
-        tracing_subscriber::fmt::format::Format,
-        BoxMakeWriter,
-    >,
+    Box<dyn tracing_subscriber::layer::Layer<Registry> + Send + Sync>,
```

Comment:
`Layer` is imported above already.

```suggestion
    Box<dyn Layer<Registry> + Send + Sync>,
```